src/components/elements/table/GenericTable.tsx

```diff
@@ -176,10 +188,48 @@ export const renderHeaderCellContents = <T extends { id: string }>(
   );
 };
 
+export const renderLinkedCellContents = <T extends { id: string }>(
```

Comment:
Can you add a header comment explaining when/why this is used? My guess is: this is used for cases where the table has a "rowLinkTo" defined. When the row is linked, we want to render an `a` inside every cell so that the whole row is clickable. Only the first one is tabbable and has a focus treatment.

Maybe rename: `renderLinkedRowCellContents`?